### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 55ritesh**

### CODE REVIEW

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
@55ritesh You are not supposed to edit the code in the README file. Revert it to its original state, and place you code in `./Tree/tree.cpp`

**Comment by 55ritesh** [comment]
changed

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
You again have created an un-extensioned file. Please rename the file to `tree.cpp`

File: Data Structures/Tree/Implementation of tree data structure [hunk 1]
```diff
@@ -0,0 +1,61 @@
+## Tree Data Structure
```
Comment (harshit-sharma-gits) [inline Data Structures/Tree/Implementation of tree data structure:1]: You also have to add your name, email in the first line as a comment in a specified format. See issue #12

**Review by 55ritesh — COMMENTED** [review]
made the changes

**Review by harshit-sharma-gits — APPROVED** [review]
Now its cool, approving.

**Pull request merged**